### PR TITLE
Text under H3 heading is larger than other text - issue fix

### PR DESCRIPTION
**Pull request by Sreejith-Websphere**
#### What was fixed?  (Issue # or description of fix)

Related to https://github.com/openliberty/openliberty.io/issues/2612

#### Were the changes tested on
- [ ] Firefox (Desktop)
- [ ] Safari (Desktop)
- [ ] Chrome (Desktop)
- [ ] Internet Explorer (Desktop)
- [ ] iOS (Mobile)
- [ ] Android (Mobile)
#### Running validation tools
- [ ] https://validator.w3.org/checklink
- [ ] https://validator.w3.org
- [ ] IBM Equal Access Accessibility Checker (https://www.ibm.com/able/toolkit/verify/automated)
- [ ] Lighthouse (in Chrome dev tools)

### CODE REVIEW

File: src/main/content/antora_ui/src/sass/nav.scss [hunk 1]
```diff
@@ -383,6 +385,7 @@ html.is-clipped--nav {
   border-radius: 7px;
   border-top: none;
   z-index: 2;
+  height: 75%;
```
Comment (kinueng) [inline src/main/content/antora_ui/src/sass/nav.scss:388]: Is this change necessary? It does not pertain to the problem area.

Comment (Sreejith-Websphere) [reply src/main/content/antora_ui/src/sass/nav.scss:388]: this file is added from drop-down version scroller, we don't need this file
